api/src/legal-documents/infrastructure/repositories/user-acceptance.repository.js

```diff
@@ -74,4 +74,9 @@ const update = async function (properties) {
   await knexConn(TABLE_NAME).where({ id }).update(data);
 };
 
-export { create, findByLegalDocumentVersionId, findByUserId, findLastForLegalDocument, update };
+const removeAllByUserId = async function ({ userId }) {
```

Comment:
Comme il n'y a qu'un seul argument, pas besoin de passer un objet (comme ce qui est fait dans la fonction `findByUserId` dans ce même repository) : 

```suggestion
const removeAllByUserId = async function (userId) {
```